components/host-dashboard/reports-section/TotalMoneyManagedSection.js

```diff
@@ -45,7 +59,85 @@ const getMoneyManagedChartAreas = (collectivesBalance, hostBalance, currency, is
   ];
 };
 
+const getChartOptions = intl => ({
+  chart: {
+    id: 'chart-host-report-money-managed',
+  },
+  stroke: {
+    curve: 'straight',
+    width: 2,
+  },
+  markers: {
+    size: 4,
+  },
+  colors: ['#46347F'],
+  xaxis: {
+    categories: [...new Array(12)].map(
+      (_, idx) => `${intl.formatDate(new Date(0, idx), { month: 'short' }).toUpperCase()}`,
+    ),
+  },
+});
+
+const totalMoneyManagedQuery = gqlV2/* GraphQL */ `
+  query TotalMoneyManagedQuery($hostSlug: String!, $dateFrom: DateTime!, $dateTo: DateTime!) {
+    host(slug: $hostSlug) {
+      id
+      hostMetricsTimeSeries(dateFrom: $dateFrom, dateTo: $dateTo, timeUnit: MONTH) {
+        totalMoneyManaged {
+          nodes {
+            date
+            amount {
+              value
+              valueInCents
+              currency
+            }
+          }
+        }
+      }
+    }
+  }
+`;
+
+const getQueryVariables = (hostSlug, year) => {
+  return {
+    hostSlug,
+    dateFrom: `${year}-01-01T00:00:00Z`,
+    dateTo: `${year}-12-31T23:59:59Z`,
+  };
+};
+
+const getSeriesFromData = (intl, timeSeries) => {
+  const dataToSeries = data => {
+    const series = new Array(12).fill(0); // = 12 months
+    data?.forEach(({ date, amount }) => (series[new Date(date).getMonth() + 1] = amount.value));
+    return series;
+  };
+
+  const totalMoneyManagedProgressNodes = get(timeSeries, 'totalMoneyManaged.nodes', []);
+  return [
+    {
+      name: intl.formatMessage({ defaultMessage: 'Total Managed Amount' }),
+      data: dataToSeries(totalMoneyManagedProgressNodes),
+    },
+  ];
+};
+
 const TotalMoneyManagedSection = ({ host, isLoading }) => {
+  const intl = useIntl();
+  const [showMoneyManagedChart, setShowMoneyManagedChart] = useState(false);
+  const yearsOptions = useMemo(() => getActiveYearsOptions(null), [null]);
```

Comment:
Why pass `null` rather than the `host` here?

Reply:
Ops, corrected this one. 👍🏽